tests/unit/tests_high_five_base.cpp

```diff
@@ -598,6 +592,63 @@ BOOST_AUTO_TEST_CASE(DataSpaceTest) {
     BOOST_CHECK_EQUAL(space.getDimensions()[1], 1);
 }
 
+BOOST_AUTO_TEST_CASE(DataSpaceVectorTest) {
+    using namespace HighFive;
+
+    // Create 1D shortcut dataspace
+    DataSpace space(7);
+
+    BOOST_CHECK_EQUAL(space.getDimensions().size(), 1);
+    BOOST_CHECK_EQUAL(space.getDimensions()[0], 7);
+
+    // Initializer list (explicit)
+    DataSpace space3({8, 9, 10});
+
+    BOOST_CHECK_EQUAL(space3.getDimensions().size(), 3);
```

Comment:
Use BOOST_CHECK_EQUAL_COLLECTIONS

Reply:
I just saw you use this in the last PR and already forgot about it... In my defense, I was inheriting code that was already there. :)